Source/msg.cpp

```diff
@@ -100,7 +104,17 @@ void PrePacket()
 				continue;
 			}
 
-			uint32_t pktSize = ParseCmd(playerId, (TCmd *)data);
+			if (playerId == std::numeric_limits<uint8_t>::max()) {

```

Comment:
`playerId > MAX_PLRS` is more flexible and saves you that assignment in the change above